src/styles/_catalog.scss

```diff
@@ -0,0 +1,20 @@
+@use 'variables';
+
+.catalog {
+  padding: 50px 40px;
+  margin: 0;
+
+  display: grid;
+  justify-content: center;
+  gap: 46px 48px;
+
+  @each $width, $columns in variables.$screenWidth {
+    @media screen and (max-width: $width) {
+      grid-template-columns: repeat($columns, 200px);
+    }
+  }
+
+  @media screen and (min-width: 1024) {
```

Comment:
The media query specifies a minimum width of 1024. Ensure that units are included, such as `1024px`, to avoid potential issues with the media query not being applied correctly.